### ISSUE TITLE
TOTP/2FA is not working properly

### ISSUE THREAD

**Issue by Korazza** [AuthMe/AuthMeReloaded#2113]
### What behaviour is observed:
I tried to enable totp authentication on my account in my server. It thought it was working, because it accepted the confirm code. But when it ask to insert the code after `/login` with `/2fa code <code>`, it never works.

### What behaviour is expected:
After `/2fa code <code>` working properly, AuthMe authorize me to login.

### Steps/models to reproduce:
- `/2fa add`
- Scan the QR Code or insert the secret code in your 2FA App
- `/2fa confirm <code>`
- `/logout`
- Try to login with 2FA

### Plugin list:
LuckPerms, SkinsRestorer, WorldEdit, PlaceholderAPI, ViaVersion, CustomJoinItems, BuycraftX, Vault, ViaBackwards, BetterAlias, WorldGuard, ProtocolLib, ViaRewind, Essentials, DeluxeMenus, EssentialsChat, ArmorStandTools, HeadDatabase, HolographicDisplays, EssentialsSpawn, BungeeTabListPlus, AuthMe, LiteBans

### Environment description
BungeeCord network, MySQL

### AuthMe build number:
AuthMeReloaded v5.6.0-SNAPSHOT (build: 2396)

### Configuration:
https://hastebin.com/xikoyoyafe.bash

**Comment by agent188** [comment]
You use google authenticator?

**Comment by Korazza** [comment]
Yes, I use Google Authenticator, I also tried with Authy but still the same problem.

**Comment by hank9999** [comment]
Hello, I have the same problem.
I tried 5.6.0-beta.1 and build2414 but it never works.

**Comment by DarkKingKaanMC** [comment]
Hi. I am also encountering this issue. Is there a fix yet?

**Comment by CmDs73** [comment]
did you found a solution? i have same problem here

**Comment by Korazza** [comment]
nope i did not

**Comment by hank9999** [comment]
> did you found a solution? i have same problem here

I use another 2fa plugin instead

**Comment by CmDs73** [comment]
Hi it's me again, i found how to resolve the problem (at least for mine)
I'm using bungeecord to link my servers
In my bungeecord plugin -> authmeBungee -> Config,
i had to whitelist /2fa or /totp command, and now it work pretty well

**Comment by Korazza** [comment]
Wow fantastic, maybe one day i'm going to try it, unfortunately i'm not playing at minecraft now, but if it works then well done!
